packet/packet.py

```diff
@@ -43,14 +43,52 @@ def sign(signer_username, freshman_username):
     return True
 
 
+def set_requirements(freshman_username, eboard=None, events=None, achieve=None):
```

Comment:
Could you rename this function to `set_essays` or something similar? The word "requirements" makes me think this is changing the list of required signatures or something.